ui/packages/catalog-ui-search/src/main/webapp/react-component/location/base.line.js

```diff
@@ -0,0 +1,27 @@
+const React = require('react');
+
+const { Units } = require('./common');
+const TextField = require('../text-field');
+
+const BaseLine = (props) => {
+  const { label, cursor, geometryKey, unitKey, widthKey } = props;
```

Comment:
✏️ Could define these as the arguments instead of having props. Ex: `({label, cursor, geometryKey, unitKey, widthKey}) => { ...}`